### DISCUSSION TITLE
User_CRUD

### DISCUSSION THREAD

**Discussion by joomoraes** [joomoraes/me-portfolio_mongodb_aspnet_web#10] [Ideas]
**the user CRUD is pretty much complete, so it can become customary for further development.**

_I'll open this thread for ideas and suggestions regarding the pre-release version made available._

<hr /><em>This discussion was created from the release <a href='https://github.com/joomoraes/me-portfolio_mongodb_aspnet_web/releases/tag/v0.1'>User_CRUD</a>.</em>

**Comment by lucasdewes** [comment]
Nice 👍🏼
